patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -1,6 +1,12 @@
 --- ../src-base/minecraft/net/minecraft/entity/Entity.java
 +++ ../src-work/minecraft/net/minecraft/entity/Entity.java
-@@ -85,7 +85,7 @@
+@@ -80,12 +80,13 @@
+ import net.minecraft.world.Teleporter;
+ import net.minecraft.world.World;
+ import net.minecraft.world.WorldServer;
++import net.minecraftforge.common.ForgeHooks;
```

Comment:
Do not use imports in patches, instead use the fully qualified name where it is used.

Reply:
Fixed in latest commit.